### DISCUSSION TITLE
Eero 6+, Firmware 7.6.0-3351

### DISCUSSION THREAD

**Discussion by szuch1** [Zoxc/crusader#80] [Wi-Fi routers and access points]
Just saw this and it seemed like it might be fun to add some data points to your collection. Running Crusader 0.3 between a Windows laptop using an Intel AX201 and a Windows desktop that's directly wired into the Eero router. SQM setting on the Eero was turned on. If there's any more information that might be nice to have just let me know!

![image](https://github.com/user-attachments/assets/23a6ae65-b482-4b65-b212-bcabde6725f2)
![eero-6plus-2ghz 2024-09-28 19 23 17](https://github.com/user-attachments/assets/334db2fe-f706-499b-9cce-5979e2a5bcef)

![image](https://github.com/user-attachments/assets/d3c5c66c-4002-44b2-baff-7719e77542c2)
![eero-6plus-5ghz 2024-09-28 19 17 38](https://github.com/user-attachments/assets/1d4d8483-3351-43a7-814c-25282cc85a6b)

**Comment by dtaht** [comment]
the  eero 5 was so much better in the LUL dept.
